File: CS/SilverlightApplication28/MainPageViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
// ...

namespace SilverlightApplication28 {
    public class MainPageViewModel {
        IEnumerable<Person> persons;

        public IEnumerable<Person> Persons {
            get {
                return persons;
            }
            set {
                persons = value;
            }
        }

        public MainPageViewModel() {
            persons = new List<Person>(new Person[] { 
                new Person("John", 25),
                new Person("Mary", 33)
            });
        }
    }
}

File: VB/SilverlightApplication28/MainPageViewModel.vb
Imports System.Collections.Generic
' ...

Namespace SilverlightApplication28
	Public Class MainPageViewModel
        Public Property Persons() As IEnumerable(Of Person)

		Public Sub New()
            Persons = New Person() {New Person("John", 25), New Person("Mary", 33)}
		End Sub
	End Class
End Namespace
